app/src/lib/git/push.ts

```diff
@@ -13,8 +13,15 @@ import { PushProgressParser, executionOptionsWithProgress } from '../progress'
 import { AuthenticationErrors } from './authentication'
 import { IRemote } from '../../models/remote'
 import { envForRemoteOperation } from './environment'
+import { enableGitTagsCreation } from '../feature-flag'
 
-export type PushOptions = {
+export interface IPushOptions {
```

Comment:
@rafeca why is this an interface now instead of a type? i'm not opposed i'm just not following 😕

Reply:
ups I changed this because I saw other options objects around the codebase using an interface, but now that I look more closely both `type` an `interface` are used similarly:

* `git grep 'interface [^\s]*Options' ':(exclude)app/test/fixtures/*' | wc -l` => 7 matches
* `git grep 'type [^\s]*Options' ':(exclude)app/test/fixtures/*' | wc -l` => 2 matches

I'm gonna remove this change from this PR and maybe revisit this in a separate PR.